### PR TITLE
Fixes layout for IE11

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- [EWL-5471: A1 | News article masthead is broken](https://issues.ama-assn.org/browse/EWL-5471)

## Description:
Fix masthead and layout in IE11

## To Test:
- run `gulp serve`
- visit http://localhost:3000/?p=pages-news
- observe this page matches https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-news

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-5471-news-article-masthead/html_report/index.html).

## Relevant Screenshots/GIFs:
screen shot 2018-10-11 at 4 57 55 pm

## Remaining Tasks
n/a

## Additional Notes
There is a corresponding PR that is a dependency
https://github.com/AmericanMedicalAssociation/ama-d8/pull/880

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by nstriedinger** [comment]
Spacing borked in IE11

screen shot 2018-10-16 at 1 11 47 pm

**Comment by designerneil** [comment]
screen shot 2018-10-16 at 2 42 11 pm

File: styleguide/source/assets/scss/01-atoms/_image.scss [hunk 1]
```diff
@@ -18,8 +18,10 @@
       @include gutter($margin-right-half...);
     }
 
-    &--right {
+    &--right {;
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/01-atoms/_image.scss]: @designerneil remove semicolon.
```suggestion
    &--right {
```

**Review by nstriedinger — CHANGES_REQUESTED** [review]
see comments

File: styleguide/source/assets/scss/01-atoms/_image.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/01-atoms/_image.scss]: good catch!

**Review by nstriedinger — APPROVED** [review]

**Pull request merged**